keyboards/arabica37/arabica37.c

```diff
@@ -0,0 +1,10 @@
+#include "arabica37.h"
+#include "ssd1306.h"
```

Comment:
To be blunt, I'd rather not see this implementation of OLED support. 

I'd rather see one of the more generalized ones, such as: https://docs.qmk.fm/#/feature_oled_driver